refactor(language_server): set `null` as a default value for `configPath`

Sysix opened the pull request:
nothing should change because nested config is disabled with `disable_nested_config`

camchenry commented on editors/vscode/client/Config.ts:
```diff
@@ -141,9 +141,9 @@ interface ConfigInterface {
+   * @default undefined
    */
-  configPath: string;
+  configPath: string | undefined;
```
Would it make sense to type this as `string | null` here as well, so we are consistent everywhere?

camchenry commented:
### Merge activity

* **Mar 28, 12:29 AM EDT**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Sysix committed:
refactor(language_server): set `null` as a default value for `configPath` (#10047)

nothing should change because nested config is disabled with `disable_nested_config`
